content/applications/sales/crm/acquire_leads/opportunities_form.rst

```diff
@@ -0,0 +1,109 @@
+  assigned based on the team's existing rules.
+- :guilabel:`Marked Fields`: use this field to alter how the form handles marked fields. The default
+  option is to treat marked fields as :guilabel:`Required`, which is the recommended setting.
+- :guilabel:`Mark Text`: choose how :guilabel:`Marked Fields` should be identified. The default
+  character is an asterisk (`*`).
+- :guilabel:`Labels Width`: use this field to alter the pixel width of the labels, if desired.
+- :guilabel:`On Success`: choose how the webpage reacts after a customer successfully submits a
+  form.
+- :guilabel:`URL`: if :guilabel:`Redirect` is selected in the :guilabel:`On Success` field, enter
+  the for the webpage where customers should be directed after successfully submitting a form.
+- :guilabel:`Visibility`: use the drop-down menu to add any visibility conditions for this field, if
+  desired.
+
+.. image:: opportunities_form/form-submitted-successfully.png
+   :align: center
+   :alt: The screen displayed after a customer successfully submits a contact form.
+
+.. important::
+   If *leads* are activated in CRM settings, selecting :guilabel:`Create an Opportunity` generates a
+   lead instead. To activate leads, navigate to :menuselection:`CRM app --> Configuration -->
+   Settings` and tick the :guilabel:`Leads` checkbox, then click :guilabel:`Save`.
+
+Customize contact form fields
+-----------------------------
+
+In addition to the settings for the form, the settings for each field can be customized as well.
+With the web editor menu still open, click into a field to open the field configuration settings.
+The following options are available to customize a field:
+
+- :guilabel:`Type`: choose a custom field option or an existing field type.
+- :guilabel:`Input Type`: determine the type of information customers should input. Available
+  options are :guilabel:`Text`, :guilabel:`Email`, :guilabel:`Telephone`, or :guilabel:`Url`. The
+  selection made in this field limits the format that customers can use when entering information.
+- :guilabel:`Label`: enter the name for the field.
+- :guilabel:`Position`: choose the way the label is aligned with the rest of the form. The label can
+  be hidden, above the field, to the left of the field, or right adjusted and closer to the field.
+- :guilabel:`Description`: slide the toggle to add a description for the field, which can provide
+  additional instructions to customers. Click under the field on the form to add the description.
+- :guilabel:`Placeholder`: enter an example to guide users how to input information where
+  formatting is important, such as a phone number or email address.
+- :guilabel:`Default Value`: enter a value to include in the form by default if the customer does
+  not provide information in the field. *It is not recommended to include a default value for
+  required fields*.
+- :guilabel:`Required`: slide the toggle to mark this field as required if it **must** be filled in
```

Comment:
I see! This is where the "marked" part I questioned earlier exists =D You could link to here if you wanted.